idl/cdc_topology_description.idl.hh

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace cdc {
+
+class stream_id final {
+    int64_t _first;
+    int64_t _second;
+};
```

Comment:
What do those first and second mean? Is it just a way to get a 128-bit ID? Why not a UUID then?

Reply:
In the future one of these two ints will be the token, and the other will be there to ensure uniqueness of streams across different generations (so it will be either a randomly generated number or simply the generation's number). Operating on such a pair of ints is IMO easier than on an UUID (when constructing/deconstructing a `stream_id`).

Reply:
It's impossible to understand when something is named _first and the other _second. You are steeped in the details and have it in your mind, but the reviewers aren't. Try to make it more accessible.

Reply:
My point is that no one needs to understand why this is a pair of ints *at the current moment*, because right now it's just that - a pair of int64_ts, or in other words, 128bit ID. *In the future* these integers are going to be given special semantics.

I'll leave a comment which says what the plans are.

Reply:
Added a comment